docs/pages/enroll-resources/machine-id/access-guides/access-guides.mdx

```diff
@@ -13,10 +13,10 @@ on how to do so.
 
 ## Resource Access
 
-- [Server Access](ssh.mdx): How to use Machine ID to access servers via SSH.
-- [Kubernetes Access](kubernetes.mdx): How to use Machine ID to access Kubernetes clusters.
-- [Database Access](databases.mdx): How to use Machine ID to access Database servers.
-- [Application Access](applications.mdx): How to use Machine ID to access Applications.
+- [Servers](ssh.mdx): How to use Machine ID to access servers via SSH.
```

Comment:
Hm - maybe this should just be "SSH". Previously "Server Access" referred to something concrete (e.g the SSH feature set), but "Servers" alone is a little too generic I think.

Reply:
Fixed in 4789543d156c0ac43e590ca1dabf072f1201f004